script.js

```diff
@@ -2,6 +2,17 @@ const userInput = document.querySelector(".userInput");
 const itemsList = document.querySelector(".itemsList");
 let itemsCounter = 0;
 
+const enterItem = () => {
+  const item = userInput.value.replace(/^\w/, (c) => c.toUpperCase());
+  const VALID_ITEM = /^[^0-9]{2,}$/;
```

Comment:
you can move this const outside the function scope. so it won't re-create every time you call the function. if you move this to global scope it'll make more sense

Reply:
fixed!